impl/client/src/main/doc-js/pentaho/context/IServer.jsdoc

```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * The `IServer` interface exposes the part of the contextual information of the Pentaho Web Client Platform
+ * The `IServer` interface exposes the part of the environmental information
```

Comment:
With the docs review, the original text changed to:
The `IServer` interface allows access to Pentaho Web Client Platform’s
contextual information that pertains to the server application.